### PR TITLE
Recommend using NVIDIA app to resolve replay indicator issue in Troubleshooting

### PR DESCRIPTION
**Pull request by Calinou**
- See https://github.com/godotengine/godot/pull/93737#issuecomment-2198746004.

### CODE REVIEW

File: tutorials/troubleshooting.rst [hunk 1]
```diff
@@ -130,6 +130,12 @@ seen on Windows, as Linux does not have support for ShadowPlay.
 To disable this overlay, press :kbd:`Alt + Z` (default shortcut for the NVIDIA overlay)
 and disable **Settings > HUD Layout > Status Indicator** in the NVIDIA overlay.
 
+Alternatively, you can install the `new NVIDIA app
+<https://www.nvidia.com/en-us/software/nvidia-app/>` which replaces GeForce
+Experience and does not suffer from this issue. Unlike GeForce Experience, the
+NVIDIA app draws the replay indicator in the corner of the screen as opposed to
+the corner of each window,.
```
Comment (aitorciki) [inline tutorials/troubleshooting.rst:137]:
```suggestion
the corner of each window.
```

**Review by mhilbrunner — APPROVED** [review]
TIL about the new NVIDIA app. Not sure if we should recommend beta software, but I think this is fine here, and the recommendation is soft enough.

**Comment by mhilbrunner** [comment]
Thank you!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9648.

**Pull request merged**